R/headers.R

```diff
@@ -37,11 +37,14 @@ print.httr2_headers <- function(x, ..., redact = TRUE) {
   invisible(x)
 }
 
-headers_redact <- function(x, redact = TRUE) {
+headers_redact <- function(x, redact = TRUE, to_redact = NULL) {
   if (!redact) {
     x
   } else {
-    list_redact(x, "Authorization", case_sensitive = FALSE)
+    to_redact <- union(attr(x, "redact"), c("Authorization", to_redact = to_redact))
```

Comment:
`Authorization` is added manually for the tests to keep working. Do you want to keep it that way to help users avoid accidentally showing credentials? Or should we remove it as it is added anyway in `req_auth_basic()` and `req_auth_bearer_token()`?

Reply:
I think as long as it's added by the calls that actually use the `Authorization` header it should be ok.